VideoAPI/VideoApi.DAL/Queries/GetInstantMessagesForConferenceQuery.cs

```diff
@@ -1,7 +1,8 @@
-﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Security.Cryptography.X509Certificates;
```

Comment:
Do we need this import? I don't see any reference to security related activities in this change